src/integrationTest/sam.test.ts

```diff
@@ -489,7 +466,15 @@ describe('SAM Integration Tests', async function () {
                 })
 
                 it('produces an error when creating a SAM Application to the same location', async function () {
-                    await assert.rejects(createSamApplication(testDir), 'Promise was not rejected')
+                    if (alreadyTested == false) {
```

Comment:
If we only need this to run once we should probably move it out of the scenario testing